app/ai/voice/agents/breeze_buddy/observers/manager.py

```
@@ -0,0 +1,132 @@
+                if not obs._detected
+                and self._turn_count >= obs.config.start_after_turn
+            ]
+            if not eligible:
+                return
+
+            results = await asyncio.gather(
+                *[obs.check(transcript) for obs in eligible],
+                return_exceptions=True,
+            )
+
+            for obs, result in zip(eligible, results):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)